$NetBSD: patch-GDALmake.opt.in,v 1.3 2017/01/13 14:48:26 gdt Exp $

Fix occasional paralell build failure:
  ogr/ogr_core.h:34:26: fatal error: gdal_version.h: No such file or directory
by ensuring we have the correct includes.

--- GDALmake.opt.in.orig	2016-07-01 10:22:10.000000000 +0000
+++ GDALmake.opt.in
@@ -66,7 +66,7 @@ INST_DOCS	=	@exec_prefix@/doc
 INST_MAN	=	@mandir@
 INST_HTML	=	$(HOME)/www/gdal
 
-CPPFLAGS	= @CPPFLAGS@ -I$(GDAL_ROOT)/port @EXTRA_INCLUDES@ -DGDAL_COMPILATION
+CPPFLAGS	= @CPPFLAGS@ -I$(GDAL_ROOT)/port -I$(GDAL_ROOT)/gcore @EXTRA_INCLUDES@ -DGDAL_COMPILATION
 CFLAGS		= @CFLAGS@ @C_WFLAGS@ $(USER_DEFS)
 CXXFLAGS	= @CXXFLAGS@ @CXX_WFLAGS@ $(USER_DEFS)
 LDFLAGS     = @LDFLAGS@
